### ISSUE TITLE
Raccoon don't see/can't download apps that are in development mode

### ISSUE THREAD

**Issue by thecoderok** [onyxbits/raccoon4#91]
Sample app: https://play.google.com/store/apps/details?id=com.coldfiregames.dungeon.manager.tycoon.idle.game&hl=en&gl=US

"This app is in development. It may be unstable."

![image](https://user-images.githubusercontent.com/3173477/113059420-47113800-9164-11eb-9ecb-dfc5f0bf7076.png)

Tested in v. 4.17.0
Exception thrown: "com.akdeniz.googleplaycrawler.GooglePlayException: The item you were attempting to purchase could not be found."

![image](https://user-images.githubusercontent.com/3173477/113059286-20eb9800-9164-11eb-9e66-ca9b7948cb57.png)

I can download this app on the phone

**Comment by jpstotz** [comment]
I don't think that the development state is the main problem that you can not download the app. 
I would assume the reason is the used device profile you use for downloading. The device profile used by Raccoon is pretty old if I remember correctly and most likely misses at least one feature that is required by this game (the game seems to require OpenGL and a 64bit device). 
Or the game is restricted to certain regions in the world.

**Comment by thecoderok** [comment]
I believe that the error is different when device is not compatible.

`com.akdeniz.googleplaycrawler.GooglePlayException: Your device is not compatible with this item.`

**Comment by jpstotz** [comment]
You are correct, usually you can query at least the app meta data even using an incompatible device. But for an unknown reason in a few cases Google sends back an "not found" result if you are trying to query the app meta data but it definitely depends on the used account respectively the used device profile, because using a different account and device profile I can successfully get the app meta data and also download the apk.

I have not figured out why it works for some devices and fails for others. As Google very often update the protobuf based API and add or removes features it is very difficult to identify the reason behind this behavior.
